cmd/trace-agent/remote_config_test.go

```diff
@@ -90,29 +90,29 @@ func TestTags(t *testing.T) {
 	}{
```

Comment:
Can you add some tests that check the results of normalization for the service and environment?